test-images/ol-7/Dockerfile

```diff
@@ -0,0 +1,73 @@
+FROM oraclelinux:7
+
+ARG CITUS_VERSION
+# Format should be XY and should not include dots e.g for 10.2.1=>102
+ARG CITUS_MAJOR_VERSION
+ARG PG_MAJOR
+ARG FANCY=1
+ARG HLL_VERSION=2.16.citus-1
+ARG TOPN_VERSION=2.4.0.citus-1
+ARG PACKAGE_RELEASE_SUFFIX=el7
+
+ENV CITUS_VERSION ${CITUS_VERSION}
+
+ENV PG_MAJOR ${PG_MAJOR}
+
+
+RUN yum update -y && \
+    yum install -y curl
+
+
+
+#RUN yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+
+RUN export CITUS_MAJOR_VER=${CITUS_MAJOR_VERSION//./} && \
+    curl https://raw.githubusercontent.com/citusdata/packaging/ol_problems/community/rpm.sh |  bash && \
```

Comment:
- [ ] After closing https://github.com/citusdata/packaging/pull/836, This should be changed

Reply:
Did you mean the commented line (21) by saying "this" here ?